ow_lander/scripts/trajectory_async_execution.py

```diff
@@ -90,8 +90,8 @@ def stop(self):
         """
         Stops the execution of the last trajectory submitted for executoin
         """
-        goal_active = self.is_active()
-        if self._connected and goal_active == 1:
+        action_state = self.get_state()
+        if self._connected and action_state == 1:
```

Comment:
A named constant instead of '1' would make this more understandable.

Reply:
Done!